Update unlock-dao-tokens.mdx (#14601)
unlock-protocol/unlock: docs/docs/governance/unlock-dao-tokens.mdx

In July 2024, the Unlock DAO approved a migration to the Base network. In order to effectively vote on the Base network, a new token was introduced: UP (Unlock Protocol Token).
UDT holders can exchange their UDT for UP on the Base network. The migration is a 1-to-1000 swap and can be conveniently achieved through a [migration website](https://up-swap.unlock-protocol.com/) in 4 steps:

- Bridging UDT to Base (from any network)
- Delegating votes on the Unlock DAO on Base
- Approving tokens to be swapped
- Swapping UDT for UP

:::info
The Unlock Foundation is offering a token incentive to perform the migration to UP. More details can be found on the [migration website](https://up-swap.unlock-protocol.com/).
:::

## UP Token Contract Information

The UP token has been deployed on Base at the following address: [0xac27fa800955849d6d17cc8952ba9dd6eaa66187](https://basescan.org/token/0xac27fa800955849d6d17cc8952ba9dd6eaa66187). It is using OpenZeppelin's ERC20 implementation and is an upgradable contract. The proxy admin is the Timelock contract on Base, which means that it can be upgraded by the Unlock DAO only.

> ## UDT Contract Addresses

Reviewer: Technically, the component below renders whatever governance token is set as the reward on per network. Right now, it is indeed UDT on all networks, but in the next few days/weeks, it will show UP for the Base network.

Maybe we keep this list and for each network we show both addresses (when applicable) UDT and UP, as well as which token is "rewarded". In that case, I will change the name of this section for something like "UDT and UP addresses per network" ? wdyt?

> ## UDT Contract Addresses

Author: ```suggestion
## UDT and UP Addresses Per Network
```

<PieChart
  series={[
    {
      data: [
        {
          value: 300000000,
          label: 'Unlock Labs (locked)',
        },
        { value: 150000000, label: 'Community (float)' },
        { value: 100000000, label: 'DAO' },
        { value: 200000000, label: 'Liquidity Pool' },
        { value: 250000000, label: 'Airdrops & Campaigns' },
      ],
    },
  ]}
  width={800}
  height={300}
/>

### Will the supply of UP change over time?

The maximum supply of the Unlock Protocol UP token is fixed at the upper bound of 1,000,000,000 tokens. Over time, the Unlock Protocol does collect a protocol fee on financial transactions that use the protocol. These fees are can be used as component of a buyback-and-burn mechanism, that may reduce the total supply of UP tokens over time.

### Are there liquidity pools for the Unlock Protocol UP token?

There is a Uniswap V3 liquidity pool for Base ETH/UP pairs at [https://app.uniswap.org/explore/pools/base/0x9EF81F4E2F2f15Ff1c0C3f8c9ECc636580025242](https://app.uniswap.org/explore/pools/base/0x9EF81F4E2F2f15Ff1c0C3f8c9ECc636580025242)

### What is the difference between the Unlock Protocol UP token (UP) and the Unlock DAO Token (UDT)

> UP token transactions are faster and have significantly lower transaction (gas) fees than UDT transactions. This is as a result of the UP token being on Base, rather than on Ethereum mainnet.

Author: ```suggestion
- UP has been deployed on Base and benefits from faster transactions while still allowing governance
- UP has 1,000,000,000 supply while UDT had a supply for 1,000,000 and UDT/UP has a fixed "exchange" rate of 1 for 1000```

- Earlier transactions yield a larger amount of UP tokens, everything else being equal; and,
- The cost of buying UP on a decentralized exchange is always lower than the cost of sending "fake" transactions to the network in order to collect UP

Another way to earn UP and join the DAO is to [apply for grants](./grants-bounties).

## Protocol Fee

Unlock Protocol includes a "fee switch” that is currently set to 1%. This value was set by the Unlock [DAO](./unlock-dao) through an [onchain proposal](https://www.tally.xyz/gov/unlock/proposal/72110981722145472193202862106710876303006911465748904597113931014096574827698) and approved in May 2024.

When the fee switch is set to any value larger than 0%, on every transaction for which there is a payment ([`purchase`](/core-protocol/smart-contracts-api/PublicLock#purchase), [`extend`](/core-protocol/smart-contracts-api/PublicLock#extend) or [`renewMembershipFor`](/core-protocol/smart-contracts-api/PublicLock#renewmembershipfor)), the lock contract will send a portion of the payment to the Unlock contract on that network. These fees are used to help sustain the decentralized protocol.

### Swap and Burn

> Current, the protocol fees collected by way of the fee switch can be "swapped and burned" by calling the `swapAndBurn` function on the Unlock contract. This function will swap the collected fees for UP (or UDT) tokens on a decentralized exchange and then burn them.

Reviewer: ```suggestion
Currently, the protocol fees collected by way of the fee switch can be "swapped and burned" by calling the `swapAndBurn` function on the Unlock contract. This function will swap the collected fees for UP (or UDT) tokens on a decentralized exchange and then burn them.
```

> ## Bridging UDT from Mainnet to Base and Exchanging the UDT for the Unlock Protocol UP Token

Reviewer: This looks a bit strange.

<img width="848" alt="Screenshot 2024-09-16 at 4 45 06 PM" src="https://github.com/user-attachments/assets/59b8147e-4d8d-4f99-99ab-2616dec71872">

Maybe we need to make this title shorter?

> ## Bridging UDT from Mainnet to Base and Exchanging the UDT for the Unlock Protocol UP Token

Author: ```suggestion
## Bridging and Exchanging UDT and UP
```

        },
        { value: 150000000, label: 'Community (float)' },
        { value: 100000000, label: 'DAO' },
        { value: 200000000, label: 'Liquidity Pool' },
        { value: 250000000, label: 'Airdrops & Campaigns' },
      ],
    },
  ]}
  width={800}
  height={300}
/>

### Will the supply of UP change over time?

The maximum supply of the Unlock Protocol UP token is fixed at the upper bound of 1,000,000,000 tokens. Over time, the Unlock Protocol does collect a protocol fee on financial transactions that use the protocol. These fees are used as component of a buyback-and-burn mechanism, that may reduce the total supply of UP tokens over time.

### Are there liquidity pools for the Unlock Protocol UP token?

There is a Uniswap V3 liquidity pool for Base ETH/UP pairs at [https://app.uniswap.org/explore/pools/base/0x9EF81F4E2F2f15Ff1c0C3f8c9ECc636580025242](https://app.uniswap.org/explore/pools/base/0x9EF81F4E2F2f15Ff1c0C3f8c9ECc636580025242)

### What is the difference between the Unlock Protocol UP token (UP) and the Unlock DAO Token (UDT)

- UP has been deployed on Base and benefits from faster transactions while still allowing governance

> - UP has 1,000,000,000 supply while UDT had a 1,000,000 supply and UDT/UP has a fixed "exchange" rate of 1 for 1000```

Author: ```suggestion
- UP has 1,000,000,000 supply while UDT had a 1,000,000 supply and UDT/UP has a fixed "exchange" rate of 1 for 1000
```

> - UP has 1,000,000,000 supply while UDT had a supply for 1,000,000 and UDT/UP has a fixed "exchange" rate of 1 for 1000```

Author: ```suggestion
- UP has 1,000,000,000 supply while UDT had a 1,000,000 supply and UDT/UP has a fixed "exchange" rate of 1 for 1000```
```

### Are UP tokens required?
UP (and UDT, its predecessor) are perfectly _optional_. Creators can deploy smart contracts without knowledge of UP and their members can similarly purchase memberships without knowing about these tokens.

### Graphic assets for the Unlock Protocol UP token

> #### Square image

Author: ```suggestion
#### Square image (Right-click image to open or download)
```

![UP Token square icon](/img/icons/UP-token-square.png)

> #### Round image

Author: ```suggestion
#### Round image (Right-click image to open or download)
```

### Swapping UDT for UP

If you have earn UDT on any network, you can swap them for UP. In order to do so, you first need to "bring" the UDT back to Base (using a bridge like [Superbridge](https://superbridge.app/base)), and then swap them for UP. The migration is a 1:1000 swap and can be conveniently achieved through a [migration website](https://up-swap.unlock-protocol.com/).

> ## What is the Unlock Protocol UP token (UP)?

Reviewer: Let's maybe add a title (## Frequently Asked Questions) above these and "downgrade" these to h3 (###)

Author: FAQ title and section added

The **UP token** (Unlock Protocol Token) is the native governance token of the Unlock Protocol.

## What is the Unlock Protocol UP token used for?

The UP token serves several key functions:

1. **Incentives**: The UP token is used to reward developers and creators who build on or integrate the Unlock Protocol. For example, developers who deploy smart contracts on the platform may earn UP tokens as rewards.
2. **Governance**: UP token holders can participate in the governance of the protocol by voting on proposals or protocol changes.

## How do you get the Unlock Protocol UP token?

> There are multiple ways for individuals to get the UP token.

Reviewer: ```suggestion
There are multiple ways for individuals or teams to get the UP token.
```

> - **Swap UDT for UP** — Holders of UDT can swap UDT for UP

Reviewer: ```suggestion
- **Swap UDT for UP** — Holders of UDT can swap UDT for UP, using [this dedicated page](https://up-swap.unlock-protocol.com/).
```

- **Purchase UP directly** — The Unlock Protocol UP token may be acquired on decentralized exchanges such as Uniswap

> - **Receive UP as grants from the Unlock DAO** — The Unlock DAO has a grants program, where builders can request grants of UP for projects built using Unlock Protocol

Reviewer: Please add link to the relevant page in the docs

Author: will add links as part of larger linking work

> - **Protocol rewards** — Transactions that use Unlock Protocol that contain a financial component can include a referrer fee, paid in the Unlock Protocol UP token

Reviewer: Please add link to relevant section above

Author: will add links as part of larger linking work

## What blockchain network is the Unlock Protocol UP token issued on?

The UP token is on the [Base](https://base.org) blockchain network.

## What is the contract address of the Unlock Protocol UP token?

The contract for the UP token is [https://basescan.org/token/0xac27fa800955849d6d17cc8952ba9dd6eaa66187#code](https://basescan.org/token/0xac27fa800955849d6d17cc8952ba9dd6eaa66187#code)

## What is the supply of the Unlock Protocol UP token?

> The UP token on Base has a supply of 1,000,000,000 (one billion) tokens.

Reviewer: ```suggestion
The UP token on Base has fixed a supply of 1,000,000,000 (one billion) tokens.
```

## What are the tokenomics of the UP token?

Holdings of the UP token are distributed across a number of different groups in the Unlock Protocol ecosystem, including:

- Unlock Labs
- Community
- DAO
- Liquidity Pools
- Airdrops & Campaigns

> TO DO: PUT PIE CHART HERE

Reviewer: right :)

Author: pie chart component added

## Will the supply of UP change over time?

> The maximum supply of the Unlock Protocol UP token is fixed at the upper bound of 1,000,000,000 tokens. Over time, the Unlock Protocol does collect a protocol fee on financial transactions that use the protocol. These fees are can be used as component of a buyback-and-burn mechanism, that may reduce the total supply of UP tokens over time.

Reviewer: ```suggestion
The maximum supply of the Unlock Protocol UP token is fixed at the upper bound of 1,000,000,000 tokens. Over time, the Unlock Protocol does collect a protocol fee on financial transactions that use the protocol. These fees are used as component of a buyback-and-burn mechanism, that may reduce the total supply of UP tokens over time.
```

## Are there liquidity pools for the Unlock Protocol UP token?

There is a Uniswap V3 liquidity pool for Base ETH/UP pairs at [https://app.uniswap.org/explore/pools/base/0x9EF81F4E2F2f15Ff1c0C3f8c9ECc636580025242](https://app.uniswap.org/explore/pools/base/0x9EF81F4E2F2f15Ff1c0C3f8c9ECc636580025242)

## What is the difference between the Unlock Protocol UP token (UP) and the Unlock DAO Token (UDT)

> UP token transactions are faster and have significantly lower transaction (gas) fees than UDT transactions. This is as a result of the UP token being on Base, rather than on Ethereum mainnet.

Reviewer: I don't think that's correct and that does not really convey the difference between UDT and UP. (Tehcnically you can swap UDT on Base "as fast" as UP on Base).

I'd say the differences are:
- "Canonical" UP has been deployed on Base and benefits from faster transactions while still allowing governance. 
- UP has 1,000,000,000 supply while UDT had a supply for 1,000,000 and UDT/UP has a fixed "exchange" rate for 1 for 1000.
Outside of this the tokens are the same and since you can always exchange one for the other, they are the same from a speculation perspective.

Author: copy updated to above bullets

## How do you bridge UDT from mainnet to Base in order to exchange the UDT for the Unlock Protocol UP token?

> In essence, the UP token on Base is an analogous version of the UDT token on mainnet. (In order to eliminate potential confusion between the two tokens, a different name — UP — was chosen on Base.)

Reviewer: I think this section should more or less replace the section `Swapping UDT for UP` above?

> In essence, the UP token on Base is an analogous version of the UDT token on mainnet. (In order to eliminate potential confusion between the two tokens, a different name — UP — was chosen on Base.)

Author: moved higher to replace other section

> *The Unlock Labs team has created a website at https://up-swap.unlock-protocol.com/ where anyone holding UDT can bridge their UDT from mainnet to Base and swap that UDT for UP governance tokens.*
> 

The Unlock Labs team has created a website at [https://up-swap.unlock-protocol.com/](https://up-swap.unlock-protocol.com/) where anyone holding UDT can bridge their UDT from mainnet to Base and swap that UDT for UP governance tokens.

You can read more about bridging UDT on mainnet to Base and exchanging bridged UDT to UP on Base [here](https://paragraph.xyz/@unlockprotocol/bridge-earn).

## Short description of the Unlock Protocol UP token

The Unlock Protocol UP token (UP) is the governance token of the Unlock Protocol, used for rewarding developers and creators, as well as for participating in governance through voting on proposals. It can be acquired by swapping UDT, purchasing it on decentralized exchanges, or earning it via protocol rewards and grants, with a fixed supply of 1 billion tokens on the Base blockchain.

## Graphic assets for the Unlock Protocol UP token

- Square image
- Round image

> 

Reviewer: This does not seem correct?

Author: added
